src/services/types.ts

```diff
@@ -71,6 +71,10 @@ namespace ts {
         getLineAndCharacterOfPosition(pos: number): LineAndCharacter;
     }
 
+    export interface SourceMapSource {
+        /* @internal */ getLineAndCharacterOfPosition(pos: number): LineAndCharacter;

```

Comment:
Although, `SourceFileLike` isn't really used in the public API anyways so maybe this doesn't matter.